EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/DependenciesPanel.java

```diff
@@ -225,6 +231,26 @@ public void onSubmit(final AjaxRequestTarget target, final Form form) {
         dependencyForm.add(addDependencyButton);
     }
 
+    private void addDeleteDependencyButton() {
+        AjaxLink deleteDependencyButton = new AjaxLink<Void>("deleteDependencyButton") {
+            private static final long serialVersionUID = 1L;
+
+            @Override
+            public void onClick(final AjaxRequestTarget target) {
+                AttributeDependency dependency = AttributeDependency.getAttributeDependencyIfExists(selectedEventType, selectedBaseAttribute, selectedDependentAttribute);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 170 characters long line (which is greater than 150 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00103)